### ISSUE TITLE
Test junit-insights in other open source projects

### ISSUE THREAD

**Issue by silasmahler** [adessoAG/junit-insights#20]
Describe which projects it was tested in and which results showed up.
Create documentation about this.

Depends on #12

**Comment by florianluediger** [comment]
List of tested projects (observed problems in parentheses):

- https://github.com/bonigarcia/mastering-junit5 (#32, #33, #34)

**Comment by silasmahler** [comment]
Works now in: https://github.com/CarloMicieli/kotlin-spring5-trenako (#42)

with build.gradle.kts ...
```kotlin
repositories {
        maven("https://dl.bintray.com/adesso/junit-insights")
}

dependencies {
    testCompile ("de.adesso:junit-insights:0.0.1")
}
````

**Comment by silasmahler** [comment]
Works now in: https://github.com/geowarin/graphql-webflux (#37)

**Comment by silasmahler** [comment]
The project: https://github.com/mvpjava/springboot-JUnit5-tutorial seems to be not working even without our library.

**Comment by florianluediger** [comment]
So since #50 is merged, all of the above mentioned small projects work correctly with our extension. However it would be nice to have a slightly larger project to test this with.

Check the status of the switch to JUnit 5 in thombergs/budgeteer#203

**Comment by florianluediger** [comment]
#57 should fix a lot of issues with other projects again, so as soon as that is merged, I think we can close this issue. If we notice any more errors when testing with other projects, separate issues should be opened with more detailed error descriptions.
